### ISSUE TITLE
npm builds broken

### ISSUE THREAD

**Issue by jjonek** [react-d3/react-d3-basics#28]
```
Error: Cannot find module '/Users/lijungchi/Documents/react-d3/react-d3-basic/node_modules/babel-preset-react-hmre/node_modules/redbox-react/lib/index.js'
```

try it in your browser: https://tonicdev.com/npm/react-d3-basic

same applies for other repos too.

**Comment by vineethawal** [comment]
+1

**Comment by BluMag** [comment]
:+1:

**Comment by chilijung** [comment]
@jjonek @vineethawal @BluMag fixed in react-d3-basic@1.6.6, thanks for reporting this issue.
